extras/cross-compile/README.md

```diff
@@ -8,51 +8,74 @@ This build configuration has only been tested with the Raspberry Pi 4, CM4, and
 
 ## Bringing up the build environment
 
-  1. Install Docker (and Docker Compose if not using Docker Desktop).
-  1. Bring up the cross-compile environment:
+1. Install Docker (and Docker Compose if not using Docker Desktop).
```

Comment:
Try to leave all numbered lists with `1. ` for all bullet points as the lists automatically increment. In practice, this reduces changelogs